src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
Additionally since we have `load()` defined as `public load<T extends Object>` and if we're not casting (want to have dynamic object we're gonna need to cast  to access any property (otherwise it will compain about missing props again, am I right @a-serdukov-mobilunity?

can we define load like `load<T = IRavenObject>(...)`?